lib/rust/parser/src/lexer.rs

```diff
@@ -714,6 +714,11 @@ fn analyze_operator(token: &str) -> token::OperatorProperties {
                 .with_unary_prefix_mode(token::Precedence::max())
                 .as_compile_time_operation()
                 .as_suspension(),
+        ".." =>
```

Comment:
Next time I should remember: Avoid the _case of_ checks in `TreeToIr` and rather do them here properly!